src/main/resources/application.yml

```diff
@@ -1,2 +1,14 @@
 server:
   port: 9000
+
+---
+spring:
+  profiles: local
```

Comment:
I like use of profiles 👍 

I'd love app to fail to start when no profile is selected in `fail fast` spirit.

Reply:
Maybe we don't need profiles... since the multi environments wont be configured inside the application

Reply:
i've decided to get rid of this for now as it's not necessary